app/assets/stylesheets/cart_pursuits.scss

```diff
@@ -14,3 +14,8 @@ body .cart_pursuits .no-items {
 .no-items a:hover {
   color: #2196f3;
 }
+
+.cart h2 {
+  font-size: 150%;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.